### ISSUE TITLE
Disable validation on `keypressed`

### ISSUE THREAD

**Issue by depsimon** [azure-ad-b2c/samples#583]
We use a custom policy and we have a frustratring behavior for the validation.

For instance, we have an email input.

Current Behavior : 

- Focus the input
- Type something 
- As soon as you typed the first letter, the validation error pops up: "Please enter a valid email address."

Expected :

- Focus the input
- Type something
- As long as you don't focus out of the input, don't trigger the validation

We don't want the user to be shown the error when he didn't have the possibility to enter a valid email adress.

This is a `selfasserted` page layout.

The issue is present in the latest version (2.1.29). I didn't try them all, but version 2.1.0 has a different behavior where it seems that the validation is only triggered when focusing out or when idle for the configured delay.

This is better but not ideal.

**Comment by gopal-amlekar** [comment]
The latest pageLayout I think is 2.1.26 but you are right, the validation occurs immediately on any keypress (even on pressing Caps lock or Ctrl keys). I am afraid to switch back to some version as a number of things change such as the error messages.

**Comment by Evertvdw** [comment]
@depsimon Did you by any chance find a solution or are we still waiting for an answer? I have the same problem now using the selfasserted layout, even in the 2.1.30 version which is currently the latest.

I can already hear the users complaining if we keep it like this.

**Comment by depsimon** [comment]
Nope, we moved to another Auth provider. There was just too many issues in the product ...

**Comment by Evertvdw** [comment]
@depsimon Thats not reassuring, but thanks for the quick feedback! I will submit a ticket referencing this issue and hopefully they will fix it.
